### ISSUE TITLE
Remove old ECR images from tooling account

### ISSUE THREAD

**Issue by jeffshep** [awslabs/aws-dataall#333]
**Is your idea related to a problem? Please describe.**
Amazon Inspector will scan and raise vulnerabilities against versions of ECR images that are not currently being used by data.all.
ECR image maintenance should be a responsibility of the data.all service.

**Describe the solution you'd like**
Automatically delete previous versions of ECR images when the tooling pipeline is run
Maintaining at least the previous version is desire-able for any CloudFormation stack rollback requirements.

**Comment by anmolsgandhi** [comment]
@petrkalos @dlpzx is this still relevant or its fixed and we can close this?

**Comment by petrkalos** [comment]
Currently there is a [lifecycle rule that keeps 200 images](https://github.com/data-dot-all/dataall/blob/479b8f3f2c0f6d4bbe6904b4e24758fc62eb37cd/deploy/stacks/ecr_stack.py#L33). I think the reqs here will be diverse, larger orgs might want to keep more for auditing purposes and smaller fewer as cost optimisation.

I think the best way to deal with it is to make the number of images to keep configurable via a context option. wdyt?

**Comment by dlpzx** [comment]
I agree, the requirements are very diverse. In the case of the original github issue purpose the idea was to reduce the number of stored images; but given that there is already a lifecycle rule in place I would fall to this default configuration. I don't think we should be investing time on this unless there is clear requirement to do so
